lib/exosuit/instance.rb

```diff
@@ -44,5 +44,14 @@ def self.all
     def self.running
       all.select { |i| i.state.name == 'running' }
     end
+
+    def self.latest_ubuntu_ami
+      ubuntu_image ||= Exosuit.ec2.images(
+          {
+              executable_users: ["all"],
+              filters: [ { name: "name",values: [IMAGE_NAME]}]
+          }
+      ).sort_by {|i| i.creation_date}.last
```

Comment:
Just a few style items (I need to get Rubocop or something on here to automate style stuff):

- Let's use single quotes in places where double quotes aren't needed
- Let's make the spacing more consistent - `[{ name: 'name', values: [IMAGE_NAME] }]`
- I believe you can actually do `sort_by(&:creation_date).last` instead of `sort_by {|i| i.creation_date}.last`